tests/ctst/features/azureArchive.feature

```diff
@@ -156,4 +157,25 @@ Feature: Azure Archive
         | versioningConfiguration | objectCount | objectSize | restoreDays |
         |           Non versioned |           2 |        100 |           5 |
         |               Versioned |           2 |        100 |           2 |
-        # |               Suspended |           2 |        100 |           2 | 
\ No newline at end of file
+        |               Suspended |           2 |        100 |           2 | 
+
+    @2.7.0
+    @PreMerge
+    @Flaky
+    @AzureArchive
+    Scenario Outline: Pause and resume archiving to azure
+    Given a "<versioningConfiguration>" bucket
+    And a transition workflow to "e2e-azure-archive" location
+    And that lifecycle is "paused" for the "e2e-azure-archive" location
```

Comment:
We might face an issue where this makes the `e2e-azure-archive` location paused while the other tests are still running, which can cause a failure of the other tests
We might think of a way to make this test able to run not in parrallels of other tests